### PR TITLE
Refine the maze environments

### PR DESCRIPTION
**Pull request by aykut-tri**
Removes the gravity and the ground collision as well as shortens the wall for the iiwa maze example

### CODE REVIEW

File: models/iiwa_sphere_collision_maze0.yml [hunk 1]
```diff
@@ -1,16 +1,13 @@
 directives:
```
Comment (JoseBarreiros-TRI) [inline models/iiwa_sphere_collision_maze0.yml:1]: for consistency, this should be name iiwa_sphere_collision_maze0_no_ground.yml

File: models/q_sys/iiwa_box_maze_xy0.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ objects:
     file: package://quasistatic_simulator/box_for_maze_xy_rotation.sdf
```
Comment (JoseBarreiros-TRI) [inline models/q_sys/iiwa_box_maze_xy0.yml:10]: for consistency, this should be named iiwa_box_maze_xy0_no_ground.

**Review by JoseBarreiros-TRI — CHANGES_REQUESTED** [review]
just a few nits.

File: models/iiwa_sphere_collision_maze0.yml (hunk 1 above)
Comment (aykut-tri) [reply models/iiwa_sphere_collision_maze0.yml:1]: done, thanks

File: models/q_sys/iiwa_box_maze_xy0.yml (hunk 2 above)
Comment (aykut-tri) [reply models/q_sys/iiwa_box_maze_xy0.yml:10]: done, thanks

**Pull request merged**